staff/models/batch.py

```diff
@@ -62,3 +63,7 @@ def fully_enrolled(self):
         if self.enrolment_set.count() >= self.capacity and self.next_enrollable_section() is None:
             return True
         return False
+
+    def weeks_to_start(self):
+        timedelta_difference_in_days = self.start_date - datetime.date.today()
```

Comment:
think no need to call the variable `timedelta_xxx`? `days_to_start_date` might be more meaningful?